bundle/tests/run_as/databricks.yml

```diff
@@ -0,0 +1,42 @@
+bundle:
+  name: "run_as"
+
+run_as:
+  service_principal_name: "my_service_principal"
+
+targets:
+  development:
+    mode: development
+    run_as:
+      user_name: "my_user_name"
+
+resources:
+  pipelines:
```

Comment:
I actually remember there's one more caveat here: Terraform doesn't allow you to list yourself as the `OWNER` if you're the person deploying the pipeline. Does that case work in your implementation?

Reply:
@lennartkats-db I added a validation in the mutator to detect this error earlier